sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerAsyncClientTest.java

```diff
@@ -1106,6 +1165,28 @@ public void recognizeCustomFormDamagedPdf(HttpClient httpClient,
             }));
     }
 
+    @Disabled("add this test when the selection mark test file is merged")
```

Comment:
Rather than keeping the tests disabled, do you want to create a small PR to add these files to the repo first?

Reply:
#16640 

Thanks